components/prism-nevod.js

```diff
@@ -0,0 +1,112 @@
+Prism.languages.nevod = {
+	'comment': /\/\/.*|(?:\/\*[\s\S]*?(?:\*\/|$))/,
+	'string': {
+		pattern: /(?:"(?:""|[^"])*"(?!")|'(?:''|[^'])*'(?!'))!?\*?/,
+		greedy: true,
+		inside: {
+			'string-attrs': /!$|!\*$|\*$/,
+		},
+	},
+	'namespace': {
+		pattern: /(@namespace\s+)[a-zA-Z0-9\-.]+(?=\s*{)/,
+		lookbehind: true,
+	},
+	'pattern': {
+		pattern: /(@pattern\s+)?#?[a-zA-Z0-9\-.]+(?:\s*(?:[(]\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]+\s*(?:,\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]*))*)[)]))?(?=\s*=)/,
+		lookbehind: true,
+		inside: {
+			'pattern-name': {
+				pattern: /^#?[a-zA-Z0-9\-.]+/,
+				alias: 'class-name',
+			},
+			'attributes': {
+				pattern: /\(.*\)/,
+				inside: {
+					'attr-name': {
+						pattern: /[a-zA-Z0-9\-.]+/,
+						alias: 'variable',
+					},
+					'punctuation': /[,()]/,
+					'operator': {
+						pattern: /~/,
+						alias: 'attr-hidden-mark',
+					},
+				},
+			},
+		},
+	},
+	'search': {
+		pattern: /(@search\s+|#)[a-zA-Z0-9\-.]+(?:\.\*)?(?=\s*;)/,
+		alias: 'function',
+		lookbehind: true,
+	},
+	'keyword': /@(?:require|namespace|pattern|search|inside|outside|having|where)\b/,
+	'standard-pattern': {
+		pattern: /\b(?:Word|Punct|Symbol|Space|LineBreak|Start|End|Alpha|AlphaNum|Num|NumAlpha|Blank|WordBreak|Any)(?:\([a-zA-Z0-9\-.,\s+]*\))?/,
+		inside: {
+			'standard-pattern-name': {
+				pattern: /^[a-zA-Z0-9\-.]+/,
+				alias: 'builtin',
+			},
+			'quantifier': {
+				pattern: /\b\d+\s*(?:\+|-\s*\d+)?(?=\W)/,
+				alias: 'number',
+			},
+			'standard-pattern-attr': {
+				pattern: /[a-zA-Z0-9\-.]+/,
+				alias: 'builtin',
+			},
+			'punctuation': /[,()]/,
+		},
+	},
+	'quantifier': {
+		pattern: /\b\d+\s*(?:\+|-\s*\d+)?(?=\W)/,
```

Comment:
1. The first `\s*` is a bit of a problem because the string ``"1\ \ \ "`` (`1` followed by 3 spaces) will be matched as `1\ \ ` (`1` followed by 2 spaces).
2. The `(?=\W)` means that there has to be a character after the quantifier. Prism should also be able to highlight small code snippets, so the end of the string should also be valid here.

(Same for the other `quantifier` pattern)

```suggestion
		pattern: /\b\d+(?:\s*\+|\s*-\s*\d+)?(?!\w)/,
```

Reply:
Fixed, thank you.